Sources/MintKit/Mint.swift

```diff
@@ -130,7 +130,7 @@ public class Mint {
         return versionsByPackage
     }
 
-    func resolvePackage(_ package: PackageReference) throws {
+    func resolvePackage(_ package: PackageReference, silent: Bool = false) throws {
```

Comment:
I'm not super happy with that solution to silent the method.. Maybe it would be cleaner to inject output. as. an optional parameter into the functions?